### DISCUSSION TITLE
向各位大侠求教，Vercel添加项目出错。

### DISCUSSION THREAD

**Discussion by bulang86** [vercel/community#6024] [Help] [answered]
### Summary

求教，在Vercel点击添加项目时显示：“未能在“节点”上执行“insertBefore”：要在其前插入新节点的节点不是此节点的子节点。”我应该如何操作？

### Example

https://vercel.com/new/luxi-bulangs-projects

### Steps to Reproduce

![screenshot-1709639926910](https://github.com/vercel/community/assets/161733077/935cb1fc-a5c4-429b-8dc3-53d193eb51ee)

**Comment by amyegan** [comment]
Hello @bulang86. Did this error happen when deploying from a [template](https://vercel.com/templates) or when [creating a project](https://vercel.com/docs/projects/overview#creating-a-project) from the dashboard?

**Reply by bulang86** [reply, marked as answer]
你好，是的。谢谢你的关心。问题解决了。原因是项目名称我没有弄好。再次感谢！
